azkaban/remote.py

```diff
@@ -153,6 +153,8 @@ def __init__(
         self.id = self.config.parser.get('session_id', key)
       except (NoOptionError, NoSectionError):
         self.id = None # No previous ID found.
+    else:
```

Comment:
Rather than have two separate cases where the ID is set to `None`, can we set it once before the `try` block and change the `except` to simply `pass`?